frontend/pages/DashboardPage/components/MdmSolutionModal/MdmSolutionModalTableConfig.tsx

```diff
@@ -48,7 +48,6 @@ export const generateSolutionsTableHeaders = (
     Header: (): JSX.Element => {
       const titleWithToolTip = (
         <TooltipWrapper
-          position="top-start"
```

Comment:
Also, why move tooltips to below for form fields? Doesn't that obscure the field and make it difficult for an end user to click into the field and start typing.

Reply:
@gillespi314 that's [my opinion](https://github.com/fleetdm/fleet/issues/18741#issuecomment-2099297270) too 

This is @rachaelshaw 's [response](https://github.com/fleetdm/fleet/issues/18741#issuecomment-2108648956)

Which I distinctly remember a conversation having them above the table headers so they don't block the table, but now I see [the comment to move all tooltips below](https://github.com/fleetdm/fleet/issues/18741#issuecomment-2111256984)

Reply:
@gillespi314 that's a problem with tooltips in form fields no matter the placement (if it's above, it covers the field above; people don't always fill things out in order). 

I think I'm going to make a feature request for an alternative, because I'm not a huge fan of using tooltips as help text and tbh I'm getting tired of debating _which_ form content we should cover up; I don't think we should obscure any of the form, and you should be able to see the help text at the same time you fill out the field.

Reply:
Oh well. For me, tooltips below the element is more disruptive the reader's flow when scanning down the page. C'est la vie.